### DISCUSSION TITLE
@PierreQuinton  can we collabrate torchquery? + torchjd?

### DISCUSSION THREAD

**Discussion by powerofaisinstudy-debug** [TorchJD/torchjd#681] [Show and tell]
Hi @PierreQuinton,

Regarding the context, I see a great opportunity for a collaboration between TorchQuery and TorchJD. My framework is designed to act as a 'stability layer' for complex Jacobian-based training.

I propose integrating TorchQuery’s Neural Healing as an optional 'Shield' within TorchJD. This would allow users to handle multi-task backward passes with much higher numerical reliability, especially on large-scale models where floating-point errors (SDC) are common.

I am happy to work on a PR (Pull Request) or a specialized bridge between our libraries to demonstrate how this prevents gradient explosions in mtl_backward. Let me know if you are open to exploring this integration!

**Comment by ValerianRey** [comment]
Hi,
I don't think your tool is specific to JD, and I don't think it's mature enough for us to add a usage example showing how to combine torchjd with torchquery. FYI we currently only do that with PyTorch Lightning, because this library is hugely used in the community. So IMO we shouldn't do that.

Thanks for understanding.
